packages/commons/core-utils/src/examples.ts

```diff
@@ -16,6 +16,10 @@ export const INT = 1;
 
 export const INT64 = 1000000;
 
+export const UINT = 1;
+
+export const UINT64 = 1;
```

Comment:
should we make these larger ?

Reply:
Yea, good catch - I'll update to the same number we use for `INT64`.